packages/contracts-bedrock/test/L2/SuperchainTokenBridge.t.sol

```diff
@@ -123,8 +123,8 @@ contract SuperchainTokenBridgeTest is CommonTest {
             abi.encodeCall(superchainTokenBridge.relayERC20, (address(superchainERC20), _sender, _to, _amount));
         _mockAndExpect(
             Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER,
-            abi.encodeCall(
-                IL2ToL2CrossDomainMessenger.sendMessage, (_chainId, address(superchainTokenBridge), _message)
+            abi.encodeWithSignature(
```

Comment:
NIT: They use more with selector on their tests
```suggestion
            abi.encodeWithSelector(
```

Reply:
`abi.encodeWithSignature` is needed in this case because the `sendMessage` function is overloaded, otherwise it won't compile complaining the method is not unique.